Source/WSTagsField.swift

```diff
@@ -8,15 +8,27 @@
 
 import UIKit
 
+public protocol WSTagsFieldDelegate {
+    func tagInputFieldValidations() -> (valid: Bool, message: String?)
+}
+
 open class WSTagsField: UIView {
     
+    open static var MIN_NUMBER_OF_CHARACTERS: Int = 1
+    open static var MAX_NUMBER_OF_CHARACTERS: Int = 255
+    open static var MIN_CHARACTER_VALIDATION_MESSAGE: String = "You need to enter at least \(WSTagsField.MIN_NUMBER_OF_CHARACTERS) character\(WSTagsField.MIN_NUMBER_OF_CHARACTERS > 1 ? "s" : ""))"
+    open static var MAX_CHARACTER_VALIDATION_MESSAGE: String = "You can only enter \(WSTagsField.MAX_NUMBER_OF_CHARACTERS) characters"
```

Comment:
i think minCharacters and maxCharacters should be configurable and even optional ... because this is based on an open source project and we will probably contribute our changes in the future ... not everyone wants this behaviour i think ... maybe only some of the tagFields should have it ... the validation message also should be configurable if in the future we add i18n? :)

ps haven't checked the PR yet, just my comments based on the discussion